clients/samples/example_hipblaslt_preference.cpp

```diff
@@ -207,6 +208,7 @@ void mat_mul_bias_activation(Tc             alpha,
                     + (bias == nullptr ? 0 : bias[i1]);
                 if(actType != ActivationType::NONE)
                     t = actFunc(t);
+                t                                    = (scaleD == nullptr ? 1.0 : scaleD[i1]);
```

Comment:
.......
Did you ever verify it?

Reply:
It weird...
Just testing at started moment.